js/bybit.js

```diff
@@ -1823,74 +2047,149 @@ module.exports = class bybit extends Exchange {
         if (limit !== undefined) {
             request['limit'] = limit; // max 200, default 200
         }
-        let method = undefined;
-        let intervalKey = 'interval';
-        let sinceKey = 'from';
-        const isUsdcSettled = market['settle'] === 'USDC';
-        if (market['spot']) {
-            method = 'publicGetSpotV3PublicQuoteKline';
-        } else if (market['contract'] && !isUsdcSettled) {
-            if (market['linear']) {
-                // linear swaps/futures
-                const methods = {
-                    'mark': 'publicGetPublicLinearMarkPriceKline',
-                    'index': 'publicGetPublicLinearIndexPriceKline',
-                    'premiumIndex': 'publicGetPublicLinearPremiumIndexKline',
-                };
-                method = this.safeValue (methods, price, 'publicGetPublicLinearKline');
-            } else {
-                // inverse swaps/ futures
-                const methods = {
-                    'mark': 'publicGetV2PublicMarkPriceKline',
-                    'index': 'publicGetV2PublicIndexPriceKline',
-                    'premiumIndex': 'publicGetV2PublicPremiumPriceKline',
-                };
-                method = this.safeValue (methods, price, 'publicGetV2PublicKlineList');
-            }
+        request['interval'] = timeframe;
+        request['from'] = sinceTimestamp;
+        const response = await this.publicGetSpotV3PublicQuoteKline (this.extend (request, params));
+        //
+        //     {
+        //         "ret_code": "0",
+        //         "ret_msg": null,
+        //         "result": [
+        //             [
+        //                 1651837620000,
+        //                 "35831.5",
+        //                 "35831.5",
+        //                 "35801.93",
+        //                 "35817.11",
+        //                 "1.23453",
+        //                 0,
+        //                 "44213.97591627",
+        //                 24,
+        //                 "0",
+        //                 "0"
+        //             ]
+        //         ],
+        //         "ext_code": null,
+        //         "ext_info": null
+        //     }
+        //
```

Comment:
Not working, spot ohlcv has `list` not  `result` 
![image](https://user-images.githubusercontent.com/43336371/201891131-2d6814db-8a1e-49ef-83cb-de39ac44090f.png)